README.md

```diff
@@ -57,55 +60,96 @@ This image only contains ModSecurity built from the code provided on the [ModSec
+* ERRORLOG - A string value indicating the location of the error log file (Default: '/var/log/apache2/error.log')
+* GROUP - A string value indicating the name (or #number) of the group to run httpd as (Default: daemon)
+* HTTPD_MAX_REQUEST_WORKERS - AN integer indicating the maximum number of connections that will be processed simultaneously (Default: 250)
+* LOGLEVEL - A string value controlling the number of messages logged to the error_log (Default: warn)
+* METRICS_ALLOW_FROM - A string indicating a range of IP adresses that can access the apache metrics (Default: '127.0.0.0/255.0.0.0 ::1/128')
+* METRICS_DENY_FROM -  A string indicating a range of IP adresses that cannot access the apache metrics (Default: All)
+* METRICSLOG - A string indicating the path of the metrics log (Default: '/dev/null combined')
+* PERFLOG - A string indicating the path of the performance log (Default: '/dev/stdout perflogjson env=write_perflog')
+* PORT - An integer value indicating the port where the webserver is listening to (Default: 80)
+* PROXY_PRESERVE_HOST - A string indicating the use of incoming Host HTTP request header for proxy request (Default: on)
+* PROXY_SSL_CA_CERT_KEY - A string indicating the path to the server PEM-encoded private key file (Default: /usr/local/apache2/conf/server.key)
+* PROXY_SSL_CA_CERT - A string indicating the path to the server PEM-encoded X.509 certificate data file or token identifier (Default: /usr/local/apache2/conf/server.crt)
+* PROXY_SSL_CHECK_PEER_NAME - A string indicating if the host name checking for remote server certificates is to be enabled (Default: on)
+* PROXY_SSL_VERIFY - A string value indicating the type of remote server Certificate verification (Default: none)
+* PROXY_SSL - A string indicating SSL Proxy Engine Operation Switch (Default: off)
+* PROXY_TIMEOUT - An intiger indicating the network timeout for proxied requests (Default: 42)
+* REMOTEIP_INT_PROXY - A string indicating the client intranet IP addresses trusted to present the RemoteIPHeader value (Default: '10.1.0.0/16')
+* SERVER_ADMIN - A string value indicating the address where problems with the server should be e-mailed (Default: root@localhost)
+* REQ_HEADER_FORWARDED_PROTO - A string indicating the transfer protocol of the initial request (Default: 'https')
+* SSL_ENGINE - A string indicating the SSL Engine Operation Switch (Default: off)
+* TIMEOUT - ApAnache integer value indicating the number of seconds before receiving and sending time out (Default: 60)
+* USER - A string value indicating the name (or #number) of the user to run httpd as (Default: daemon)
+
+### Nginx ENV Variables
+
+* ACCESSLOG - A string value indicating the location of the access log file (Default: '/var/log/nginx/access.log')
+* BACKEND - A string indicating the partial URL for the remote server of the `proxy_pass` directive (Default: http://localhost:80)
+* BACKEND_WS - A string indicating the IP/URL of the WebSocket service (Default: ws://localhost:8080)
```

Comment:
I just noticed that we're not using this yet with Nginx. Why was it necessary to use a separate variable for web socket backends, @srueg?

Reply:
This is due to how Apache handles websocket (and especially WebSocket via TLS). It is not required on nginx.

Reply:
I mentioned this as "unused" in the README for Nginx.